pkg/volume/volume.go

```diff
@@ -103,3 +113,31 @@ func RenameDirectory(oldPath, newName string) (string, error) {
 	}
 	return newPath, nil
 }
+
+// ReadWriteModeFor returns the ReadWriteMode represented by the given boolean.
+// A value of true maps to ReadOnly, false to ReadWrite
+func ReadWriteModeFor(ro bool) ReadWriteMode {
+	if ro {
+		return ReadOnly
+	}
+
+	return ReadWrite
+}
+
+func IsReadOnlyToPod(mode ReadWriteMode) bool {
```

Comment:
Question: Does the volumes plugin really have to understand pods and nodes? Can this policy be scoped at the kubelet level? Ideally, we can use these plugins even outside of the kubelet context right?